TheWarWithin/PriestShadow.lua

```diff
@@ -2392,6 +2412,35 @@ spec:RegisterAbilities( {
             if talent.dark_evangelism.enabled then addStack( "dark_evangelism" ) end
         end,
     },
+
+    -- Add racial abilities
+    arcane_torrent = {
```

Comment:
These belong in Hekili/Classes.lua (unless there is a functional distinction from what is there).  That may be the case for Arcane Torrent but probably not Light's Judgment.  Both should probably be on the Cooldowns toggle, as well.